requirements/constraints.txt

```diff
@@ -9,9 +9,10 @@
 # linking to it here is good.
 
 
-# Keep same platform version
+# Common constraints for Open edX repos
+-c https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt
+
 celery>=5.2.2,<6.0.0
 djangorestframework<=3.14.0
-Django<5.0
 openedx-events<=9.0.0
```

Comment:
I think we need to review these constraints. Are they up to date? Are they still necessary? Also, can we add a comment next to each constrain so we know why they're there in the future?

Reply:
Did we review these constraints as I suggested above? Let me know!

Reply:
I already updated what I found in the Open edX repo, it is discussed in the team that if necessary a task is created to validate all the constraints